example/hello_classification_electron/utils.js

```diff
@@ -93,6 +92,26 @@ let singleModelTable =
       return allFormats;
     }
 
+let constructDevice =
```

Comment:
I'd prefer to naming it `configureDevice`. And please put a brief document for this function and note that the first device in the device array will be used.

Reply:
Do I need to create a new document at the path `doc\`? Or I just add some introduction in the `READMe.md` in the example path `example\hello_classification_electron\README.md`.

Reply:
doc here means comments besides code.